action.yml

```diff
@@ -293,6 +293,16 @@ runs:
           done <<< "${{ inputs.gitops-dev }}"
         fi
 
+    - name: Emit Image Build Event to Upwind.io
```

Comment:
Could we ensure that the action not fails when this step is not working?

Reply:
@0x46616c6b See https://github.com/Staffbase/gitops-github-action/pull/101